### DISCUSSION TITLE
Notification count shows in navigation menu after dialog closed

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#5854] [Feature Requests]
**Describe the bug**
The count of notifications in the side menu is visible after notifications have been opened and closed.

The number shown in the navigation menu should only reflect new, unread messages, not the total. Number should disappear when notifications dialog is closed. 

**Screenshots**

https://github.com/specify/specify7/assets/135047322/ee7d73a7-8345-4388-87f1-3053645754a3

**Comment by maxpatiiuk** [comment]
I belive that number always reflected the total number of notifications rather than unread notifications - we could change it, but I don't think current behavior is a bug

in fact, at the moment front-end is not optimized for users with a lot of notifications (see https://github.com/specify/specify7/issues/2606)
